src/bundle/Resources/config/rest.yaml

```
@@ -8,4 +8,4 @@ services:
         arguments:
             - '@ezrichtext.converter.edit.xhtml5'
         tags:
-            - { name: ezplatform.field_type.rest.processor, alias: ezrichtext }
+            - { name: ibexa.rest.field_type.processor, alias: ezrichtext }
```

This can be replaced with

```suggestion
            - { name: !php/const EzSystems\EzPlatformRestBundle\DependencyInjection\Compiler\FieldTypeProcessorPass::FIELD_TYPE_PROCESSOR_SERVICE_TAG, alias: ezrichtext }
```

but as said in the other PR, it's debatable and nitpicky I guess?